espnet/nets/pytorch_backend/nets_utils.py

```diff
@@ -160,16 +180,18 @@ def make_pad_mask(lengths, xs=None, length_dim=-1, maxlen=None):
         else:
             maxlen = xs.size(length_dim)
     else:
-        assert xs is None
-        assert maxlen >= int(max(lengths))
+        assert xs is None, "When maxlen is specified, xs must not be specified."
```

Comment:
Sorry, xs must be specified or xs must not be specified?
I want to confirm it.

Reply:
"`xs` must not be specified" is correct.